src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPageResources.resx

```diff
@@ -279,4 +279,28 @@
   <data name="JSWebView2DebuggingAdditionalText" xml:space="preserve">
     <value>(Requires the JavaScript Diagnostics component)</value>
   </data>
+  <data name="ImplicitUsingEditor_AddUsing" xml:space="preserve">
+    <value>Using</value>
+  </data>
```

Comment:
It would be very useful to have comments for each of these explaining what the text means and/or where it is used. Our translators generally see these terms in isolation and the more context we can provide, the better the translation.